build_docs.pl

```diff
@@ -374,6 +377,9 @@ sub check_kibana_links {
                 $path =~ s!\$\{PLUGIN_DOCS\}!en/elasticsearch/plugins/$version/!;
                 $path =~ s!\$\{FLEET_DOCS\}!en/fleet/$version/!;
                 $path =~ s!\$\{APM_DOCS\}!en/apm/!;
```

Comment:
It doesn't impact this PR, but do you know why `APM_DOCS` is the only path that excludes `$version`?

Reply:
I'm not sure why that was done, but it matches what's in the doc link service: https://github.com/elastic/kibana/blob/9eabc926e436e31318bcf720f270ac00e3075c60/src/core/public/doc_links/doc_links_service.ts#L30

Reply:
I guess the moral is that if you add a lot more APM-related URLS to the doc link service and decide to change the variable, you'll have to circle back and change it here too.

Reply:
👍 Thanks!